### ISSUE TITLE
DM notifs did not notify me of large rob

### ISSUE THREAD

**Issue by HongKonger1234** [DankMemer/dank-memer-suggestions-and-bugs#606]
**Describe the bug**
I woke up one morning to find that all the coins in my wallet (238,417 total) were gone. I had DM notifs on and checked the DM. Nothing. It was only after I tried turning off DM notifs and checking did I find out that I was robbed. Later the same day, I was notified of another robbing (of 14,393 coins), even after I did nothing. 

**To Reproduce**
1. Get robbed
May not work every time.

**Expected behavior**
It notifies me when I get robbed.

**Screenshots**
![Screenshot 2020-06-04 at 5 10 40 PM](https://user-images.githubusercontent.com/66413779/83738144-7f500300-a686-11ea-8372-5c23ac446521.png)
![Screenshot 2020-06-04 at 5 11 34 PM](https://user-images.githubusercontent.com/66413779/83738149-80813000-a686-11ea-980f-48aef2650c45.png)

**Additional context**
If you're feeling generous, I'm HongKonger#8430 (nickname Freedom to Hong Kong) in the Dank Memer Official server ;-)

**Comment by oliverysfx** [comment]
would you please explain how you are checking the notifs with dm notifs on?

**Comment by HongKonger1234** [comment]
I temporarily disabled DM notifs, checked and enabled them again.

**Comment by oliverysfx** [comment]
I see no proof that only that notif was skipped
are u sure u didn't accidently disable it?

**Comment by HongKonger1234** [comment]
Yup, I tried pls notifs that morning to see if DM notifs was disabled. It came back with a "You have DM Notifications enabled via pls settings." Unfortunately I've already deleted that message.

**Comment by oliverysfx** [comment]
When was the rob, ie the date? send a screenshot here with pls notifs view[which ever number that notif is at now]

**Comment by HongKonger1234** [comment]
![Screenshot 2020-06-04 at 7 57 59 PM](https://user-images.githubusercontent.com/66413779/83753928-bb429280-a69d-11ea-80af-26e4aa6c6be0.png)

**Comment by oliverysfx** [comment]
hmm that's a whole day of no notifs though, most likely u turned it off or smth liddat

**Comment by HongKonger1234** [comment]
Btw- if it helps, I had participated in the Dank Memer Official giveaways that night, got kicked out due to server outage, and when they were done I went straight to sleep. Didn't touch the notifications or settings. Next morning the first command I ran was pls pet feed and it said I had no coins. 

Actually, I just had a look and apparently I haven't deleted the messages from that morning.
![Screenshot 2020-06-04 at 8 07 04 PM](https://user-images.githubusercontent.com/66413779/83754712-027d5300-a69f-11ea-9e85-928b348c83f6.png)
